### ISSUE TITLE
rows_merge()

### ISSUE THREAD

**Issue by ndzuris** [rstudio/gt#691]
Please briefly describe your problem or your ideas for enhancement in the template below.

Have a question? Please don't use this form but instead ask on [RStudio community](https://community.rstudio.com), on [Stack Overflow](https://stackoverflow.com), or on Twitter (using the `#rstats` hashtag).

When describing a bug, please include a minimal reproducible example (otherwise known as a [reprex](http://reprex.tidyverse.org/)).

When posting, delete these explanatory lines and submit only the text derived from the template below.

---

Hello,

Please include a function to merge rows.  Like that requested here https://community.rstudio.com/t/merge-cells-in-a-gt-table/46770

Also, please include a function to wrap long column titles/headers.  

Without these two functions I can't really use the table I've created - it's not polished looking.  

Thank you for this package! I have been waiting a long time for this in R :) 

```r
<insert reprex here>
```

**Comment by trizniak** [comment]
It would be nice to be able to merge cells in a column based on values in a different column, similar to [flextable::merge_v](https://davidgohel.github.io/flextable/reference/merge_v.html)

**Comment by rich-iannone** [comment]
I totally agree that cell merging would be a great idea. Thanks for the suggestion!

**Comment by hjia222** [comment]
Is this been tackled yet? it will be a really helpful  function.

**Comment by bacinf12** [comment]
Can someone give an update on the status of when this might be included? Thank you kindly!

**Comment by matschmitz** [comment]
This would be a much needed feature! It's the only reason that's preventing me switching from `flextable` to `gt`.
